src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -5787,18 +4991,89 @@ void DownInterpolate4HistoryValues(Real64 const OldTimeStep,
     }
 }
 
+Real64 DownInterpolate4HistoryValues(Real64 OldTimeStep, Real64 NewTimeStep, std::array<Real64, 4> const &oldVals, std::array<Real64, 4> &newVals)
```

Comment:
Some of the temperature and humidity data are now arrays within the heat balance struct so there's a new flavor of this general interpolation function to handle those more cleanly.